### PR TITLE
Teams

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by MarkWoulfeNeo4j** [comment]
[Document Aura Enterprise feature - Teams [Phase 1]](https://trello.com/c/wDiEO1Yr/155-document-aura-enterprise-feature-teams-phase-1)

File: modules/ROOT/pages/platform/user-management.adoc [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+label:AuraDB-Enterprise[] label:AuraDS-Enterprise[]
+
+User management is a feature within Aura that allows you to invite users and set their roles within an isolated environment.
+
+== Tenants
+
+Tenants are the primary mechanism for granting users access to an Aura environment.
+
+The tenant you're currently viewing is displayed in the header of the Console. You can select the tenant name to open the **Tenant Dropdown** menu, which allows you to view the _Tenant ID_.
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/platform/user-management.adoc:13]:
```suggestion
The tenant you're currently viewing is displayed in the header of the Console. 
You can select the tenant name to open the **Tenant Dropdown** menu, which allows you to view the _Tenant ID_.
```

File: modules/ROOT/pages/platform/user-management.adoc [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+It is possible to be a member of multiple tenants, in which case the **Tenant Dropdown** will also display the other tenants you have access to. You can switch between these tenants by clicking on the individual tenant from the **Tenant Dropdown**.
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/platform/user-management.adoc:15]:
```suggestion
It is possible to be a member of multiple tenants, in which case the **Tenant Dropdown** will also display the other tenants you have access to. 
You can switch between these tenants by clicking on the individual tenant from the **Tenant Dropdown**.
```

File: modules/ROOT/pages/platform/user-management.adoc [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+== Team Members
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/platform/user-management.adoc:17]:
```suggestion
== Team members
```

File: modules/ROOT/pages/platform/user-management.adoc [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+Each tenant consists of a single team that allows multiple users with individual accounts to access the same environment.
+
+The team and its members can be viewed and managed from the **Team Members** page. You can access the **Team Members** page by selecting **Team Members** from the sidebar menu of the Console.
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/platform/user-management.adoc:21]:
```suggestion
The team and its members can be viewed and managed from the **Team Members** page. 
You can access the **Team Members** page by selecting **Team Members** from the sidebar menu of the Console.
```

File: modules/ROOT/pages/platform/user-management.adoc [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+=== Deleting team members
+
+As an _Admin_, to delete an existing team member from the team:
+
+. Select the **Delete member** trash can icon next to the team member's name from the **Team Members** page.
+. Select **Delete**.
+
+[NOTE]
+====
+Deleting a team member whose **Status** is _Pending invite_ is also possible by selecting the **Delete member** trash can icon next to the team member's name and then selecting **Revoke**.
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/platform/user-management.adoc:71]:
```suggestion
It is also possible to delete a team member whose **Status** is _Pending invite_.

Select the **Delete member** trash can icon next to the team member's name, and then select **Revoke**.
```

File: modules/ROOT/pages/platform/user-management.adoc [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+====
+
+=== Accepting an invite
+
+When invited to a team, a **Tenant invitation** modal will appear the next time you reload the Console. You can select the tenant(s) you have been invited to and choose to accept or decline the invite(s).
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/platform/user-management.adoc:76]:
```suggestion
When invited to a team, a **Tenant invitation** modal will appear the next time you reload the Console. 
You can select the tenant(s) you have been invited to and choose to accept or decline the invite(s). 
```

**Review by davidoliverSP2 — APPROVED** [review]
Looks good, thanks @MarkWoulfeNeo4j 

Some suggestions to consider 😜

File: modules/ROOT/pages/platform/user-management.adoc [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+As an _Admin_, to delete an existing team member from the team:
+
+. Select the **Delete member** trash can icon next to the team member's name from the **Team Members** page.
+. Select **Delete**.
+
+[NOTE]
+====
+It is also possible to delete a team member whose **Status** is _Pending invite_.
+
```
Comment (MarkWoulfeNeo4j) [inline modules/ROOT/pages/platform/user-management.adoc:75]:
```suggestion
```

**Pull request merged**